test/foundry-tests/PoolManager.t.sol

```diff
@@ -258,6 +267,27 @@ contract PoolManagerTest is Test, Deployers, TokenFixture, GasSnapshot, IERC1155
         manager.initialize(key, sqrtPriceX96, ZERO_BYTES);
     }
 
+    function test_initialize_revertsWithWrappedNative(address currencyAdd) public {
+        uint160 sqrtPriceX96 = 0;
+
+        Currency currency = Currency.wrap(currencyAdd);
+        Currency wrappedCurrency = Currency.wrap(wrappedNative);
+
+        // currency1 just a placeholder until the currencies are set below
+        PoolKey memory key =
+            PoolKey({currency0: currency1, currency1: currency1, fee: 3000, hooks: IHooks(address(0)), tickSpacing: 60});
+
+        (key.currency0, key.currency1) =
+            (currencyAdd < wrappedNative) ? (currency, wrappedCurrency) : (wrappedCurrency, currency);
+
+        if (currencyAdd == wrappedNative) {
+            vm.expectRevert(IPoolManager.CurrenciesInitializedOutOfOrder.selector);
```

Comment:
A bit random but reading this test made me realize maybe there should be a different error for when you try to initialize the pool with the same currencies?

Reply:
yeeeeah i did it just bc its cheaper gas-wise for `initialize` to just have 1 if statement, and itll still cause people to look at the right line. But it is a bit odd
I could change `CurrenciesInitializedOutOfOrder` to like `CurrenciesOutOfOrderOrIdentical` or something?